spec.md

```diff
@@ -726,7 +726,9 @@ being created on the Service Broker.
 
 The frequency and maximum duration of polling MAY vary by platform client. If
 a platform has a max polling duration and this limit is reached, the platform
-MUST cease polling and the operation state MUST be considered `failed`.
+MUST cease polling, the operation state MUST be considered `failed`, and the
+Platform SHOULD send a deprovision request to the Service Broker to prevent an
```

Comment:
I'd prefer if we do what you did down below and just say "SHOULD initiate orphan mitigation" with a link to that section. Then we're not duplicating what orphan mitigation means in 2 places.